### PR TITLE
Feature/je 4 idetyfikacjaslow kluczowych po serii pytan

### PR DESCRIPTION
**Pull request by kkuczynska**
sprawdź i zaakceptuj jeśli jest ok :)

### CODE REVIEW

File: src/main/java/com/jbd/Questions.java [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+import com.jbd.KeywordsFinder.Keywords;
+
+import static com.jbd.SearchCriteria.*;
+
+public class Questions {
+
+    public static final String ANSWER_POSITIVE = "1";
+    public static final String ANSWER_NEGATIVE = "0";
+    private static String EMAIL_QUESTION = "Provide email address (xxxx@xx.xx)";
+    private static String KEYWORD_QUESTION = "Provide createAndDisplayKeywordsHashSet to find in email subject. Separate them with commas.";
```
Comment (Cinek1812) [inline src/main/java/com/jbd/Questions.java]: createAndDisplayKeywordsHashSet -> it should be "Keywords" ? :)

File: src/main/resources/KeywordsForm.json [hunk 2]
```diff
@@ -1,91 +1,98 @@
 {
   "questions": [
-    "Czy email jest pilny?",
-    "Czy email jest służbowy?",
-    "Czy email dotyczy rodziny?",
-    "Czy email dotyczy spotkania?"
+    "Are you looking for an urgent email?",
```
Comment (Cinek1812) [inline src/main/resources/KeywordsForm.json]: What response do we expect ? "yes", "y" ?

Comment (Cinek1812) [inline src/main/resources/KeywordsForm.json]: We need to add mechanism that don't allow us to put words other then "y" or "n" (or"yes" "no" etc. ).
We cannot add word like "kogut" itp :)

Comment (dzefrej0) [inline src/main/resources/KeywordsForm.json]: Just what said our SM, all else is nice I think.

**Pull request merged**